Assets/Tests/InputSystem/Plugins/MockKeyboardWithUITests.cs

```diff
@@ -0,0 +1,105 @@
+using System;
+using System.Collections;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.InputSystem;
+using UnityEngine.InputSystem.UI;
+using UnityEngine.TestTools;
+using UnityEngine.UI;
+
+[Description("Check https://jira.unity3d.com/browse/ISXB-1561 for more details.")]
+public class MockKeyboardWithUITests : InputTestFixture
+{
+    private Keyboard m_Keyboard;
+    private GameObject m_CanvasGo;
+    private GameObject m_EventSystemGo;
+    private GameObject m_ButtonGo;
+    private InputActionAsset m_InputActionAsset;
+
+    public class CancellationHandler : MonoBehaviour, ICancelHandler
+    {
+        public Action trigger;
+        public void OnCancel(BaseEventData eventData) => trigger();
```

Comment:
This stuff is only needed so that I could use a lambda closure with assertion below.